Resources/Prototypes/_DV/Recipes/Cooking/Microwave/moth.yml

```diff
@@ -0,0 +1,206 @@
+# Most of these dont sound remotely moth-edible but they all have Moth in their ids???
+
+# Soups & Stews
+
+- type: microwaveMealRecipe
+  id: RecipeMothCottonSoup
+  name: cotton soup recipe
+  result: FoodMothCottonSoup
+  time: 10
+  group: Soup
```

Comment:
i usually give precedence to "moth" over the other food categories, especially if it's not edible for non-moths

Reply:
i forgot it had cotton in it real

Reply:
that goes for the other moth foods in this file(s)

Reply:
the other ones have normal ingredients and besides maybe mothic salad can be eaten by most people